src/content/docs/iast/applications.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: IAST Tests & Applications
+tags:
+    - IAST
+    - New Relic IAST
+    - Applications
+metaDescription: Find out the total number of exploitable applications and safe applications you have.
+freshnessValidatedDate: never
+---
+
+import iastCoverageSummary from 'images/iast_screenshot-crop_coverage-summary.webp'
+
+import iastEnableIast from 'images/iast_screenshot-crop_enable-iast.webp'
+
+
+The applications page informs you about the total number of found vulnerabilities in your tested applications and the number of untested applications you have. You can always add untested applications to the New Relic IAST capability. Each application test is a build of an application. Each time your pre-production environment deploys a new build and exercises a new test, it will show up with that application name. For each test run we show the application name, account, status, exploit count, test efficiency, and the last time the application was tested. You can sort by application name to see all test runs of an application together to quickly compare their status.
```

Comment:
I think you can combine sentence 2 and sentence 3 here with a ", and" to slightly improve the flow.

In sentence 5, there should be a comma after "For each test run"